src/main/java/com/github/havardh/javaflow/phases/parser/java/JavaParserFlags.java

```diff
@@ -0,0 +1,43 @@
+package com.github.havardh.javaflow.phases.parser.java;
+
+public class JavaParserFlags {
+  private boolean includeStaticFields;
+
+  private JavaParserFlags(boolean includeStaticFields) {
+    this.includeStaticFields = includeStaticFields;
+  }
+
+  public static JavaParserFlags defaultFlags() {
+    return Config.withDefaults().build();
+  }
+
+  public boolean shouldIncludeStaticFields() {
+    return includeStaticFields;
+  }
+
+  public static Config config() {
+    return new Config();
+  }
+
+  public static class Config {
```

Comment:
Where is this class needed? Couldn't we just have JavaParserFlags, without Config?

Reply:
Well actually it should be renamed to Builder. It is only used when constructing the flags object. :) This let's the flags object be immutable without having a constructor with a growing number of Boolean flags.